atomic_reactor/build.py

```diff
@@ -187,15 +192,23 @@ def set_df_path(self, path):
         self.set_base_image(base)
         logger.debug("base image specified in dockerfile = '%s'", self.base_image)
         self.parent_images.clear()
+        custom_base_images = set()
         for image in dfp.parent_images:
             image_name = ImageName.parse(image)
             if base_image_is_scratch(image_name.get_repo()):
                 image_name.tag = None
                 self.parents_ordered.append(image_name.to_str())
                 continue
+            if base_image_is_custom(image_name.to_str()):
```

Comment:
just nitpick, because `image_name.to_str()` is used 4 times in the while cycle, couldn't be better to convert it once into extra var and use that?